Project.toml

```
@@ -14,10 +14,10 @@ StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
 [compat]
 FileIO = "1.2.3"
 ImageCore = "0.8.1, 0.9"
-ProtoBuf = "0.10, 0.11"
+ProtoBuf = "0.10, 0.11, 1.0.11"
 Requires = "0.5, 1"
-StatsBase = "0.27, 0.28, 0.29, 0.30, 0.31, 0.32, 0.33"
-julia = "1.3"
+StatsBase = "0.27, 0.28, 0.29, 0.30, 0.31, 0.32, 0.33, 0.34"
+julia = "1.3, 1.9"
```

This means the same as
```suggestion
julia = "1.3"
```
because `1.3` means `[1.3.0, 2.0)` (inclusive, exclusive bounds)


But if the oldest thing we are running CI in 1.6 then lets say
```suggestion
julia = "1.6"
```